### PR TITLE
Prompt to update chrome

### PR DESCRIPTION
**Pull request by bhollis**
Take a look and see if you want to update the copy

### CODE REVIEW

File: app/scripts/dimApp.config.js [hunk 1]
```diff
@@ -49,6 +49,15 @@
           SyncService.init();
         };
 
+        var chromeVersion = /Chrome\/(\d+)/.exec($window.navigator.userAgent);
+        if (chromeVersion && chromeVersion.length === 2 && parseInt(chromeVersion[1], 10) < 50) {
```
Comment (kyleshay) [inline app/scripts/dimApp.config.js]: what did v50 introduce?

Comment (SunburnedGoose) [inline app/scripts/dimApp.config.js]: Dunno, just seemed like a round number ;)

Comment (kyleshay) [inline app/scripts/dimApp.config.js]: arrow functions: v45.0
spread operator: v46.0

Comment (SunburnedGoose) [inline app/scripts/dimApp.config.js]: Support over time.  Big gains recently.

screen shot 2016-07-10 at 12 02 55 pm

Comment (kyleshay) [inline app/scripts/dimApp.config.js]: fair enough. it's not like someone with chrome 44 would only be able to update to chrome 48 or something anyway, they'd bump to latest. just getting the message out there to update should be enough motivation.

**Comment by kyleshay** [comment]
Maybe we could provide a link to our twitter/reddit if they have issues with updating? Would linking to a help page about how to update be useful?

**Comment by SunburnedGoose** [comment]
We should have those links in high-profile locations on the UI that are just simple anchor tags in case the app breaks.

**Comment by bhollis** [comment]
Good idea.

We can do 51 since it's the latest. We need CSS variables, spread/rest, and
arrow functions for now, but lots more later.

**Pull request merged**